kotlin-website.py

```diff
@@ -233,6 +238,13 @@ def kotlin_docs_pdf():
 def community_page():
     return render_template('pages/community.html')
 
+@app.route('/user-groups/user-group-list.html')
+def user_group_list():
+    return render_template(
+        'pages/user-groups/user-group-list.html',
+        user_groups_data=site_data['user-groups'],
+        number_of_groups=sum(map(lambda section: len(section['groups']), site_data['user-groups'])))
```

Comment:
I don't know how to perform such nested summing in Jinja2, so I decided to pass another template argument. If you know a way to move it to the template, I'm happy to do follow your hint.

Reply:
I think you have chosen the right way. It looks clear.